LX.StaffScheduler.Api/Controllers/WeatherForecastController.cs

```diff
@@ -1,33 +0,0 @@
-using Microsoft.AspNetCore.Mvc;
-
-namespace LX.StaffScheduler.Api.Controllers
-{
-    [ApiController]
-    [Route("[controller]")]
-    public class WeatherForecastController : ControllerBase
-    {
-        private static readonly string[] Summaries = new[]
-        {
-            "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
-        };
-
-        private readonly ILogger<WeatherForecastController> _logger;
-
-        public WeatherForecastController(ILogger<WeatherForecastController> logger)
-        {
-            _logger = logger;
-        }
-
-        [HttpGet(Name = "GetWeatherForecast")]
-        public IEnumerable<WeatherForecast> Get()
```

Comment:
In assembly you have a file of class WeatherForecast so please remove it along with unnecessary controller